app/models/course/assessment/submission.rb

```diff
@@ -6,7 +6,8 @@ class Course::Assessment::Submission < ActiveRecord::Base
   include Course::Assessment::Submission::ExperiencePointsDisplayConcern
 
   after_save :auto_grade_submission, if: :submitted?
-  after_create :send_notification
+  after_save :send_submit_notification, if: :submitted?
```

Comment:
Good catch! @custode we should not send notifications for auto graded assessments.